style.css

```diff
@@ -2854,3 +2848,113 @@ h4:hover .learn-more, h5:hover .learn-more, h6:hover .learn-more {
     color: #ff6760;
     font-size: 75%;
 }
+
+/* Fancybox */
```

Comment:
Honestly, I don't remember.  When I first added it, I didn't want to "mess up" any of the existing styles, so I copied them to the end.

I need to go back over everything and compare it side-by-side.  Most of it *is* probably identical to what's above. :sweat_smile:

For that matter, once we switched to the carousel... the "standalone" concept didn't really get enough attention.

I'll make a note to look it all over.